tests/test_csc.py

```diff
@@ -1,4 +1,4 @@
-# This file is part of ts_hexapod.
+# This file is part of ts_mthexapod.
 #
 # Developed for the LSST Data Management System.
```

Comment:
Copyright statements referring to the LSST Data Management System should be updated to refer to the LSST Telescope and Site Systems but that can be done at a later time.

Reply:
Interesting. Yes. I'll fix it now.

Reply:
If I can figure out what to fix it to. I'm pretty sure LSST is not the right thing. I've asked on the ts software Slack channel, but not response yet.

Reply:
At this point, getting no answer, I think I'll hold off for another ticket, as you suggested.